net/http/authz/authorizer.go

```diff
@@ -69,6 +74,35 @@ func accessTokenGuardData(grant *Grant) string {
 	return v.ID
 }
 
+func x509GuardData(grant *Grant) pkix.Name {
+	var v struct {
+		Subject struct {
+			CommonName         string   `json:"cn"`
+			OrganizationalUnit []string `json:"ou"`
```

Comment:
Can we leave a todo comment here about this mapping json to pkix.Name? We need some more comprehensive plan for it. The `cn` and `ou` thing were just ad hoc examples in the docs. I am pretty sure the spec did not intend for that to be definitive.

cc @jeffomatic @tonychain